samples/go/photo-index/main.go

```diff
@@ -27,21 +29,13 @@ func index() (win bool) {
 	var dbStr = flag.String("db", "", "input database spec")
 	var outDSStr = flag.String("out-ds", "", "output dataset to write to - if empty, defaults to input dataset")
 	var parallelism = flag.Int("parallelism", 16, "number of parallel goroutines to search")
+	verbose.RegisterVerboseFlags(flag.CommandLine)
 
 	flag.Usage = usage
 	flag.Parse(false)
 
-	if flag.NArg() == 0 {
-		flag.Usage()
-		return
-	}
-
-	if flag.NArg() == 0 {
```

Comment:
got it. Yeah that's tricky, if you want to keep the behavior the same you'd need the `--db` argument, so the value would be that you could say `--db default` or `--db photos` or whatever, instead of `--db http://demo.noms.io/...`.

Because otherwise it seems ambiguous. There can be an arbitrary number of arguments after the db and out-ds flags which are the datasets, inside db, to index.